test/production/reading-request-body-in-middleware/index.test.ts

```diff
@@ -0,0 +1,108 @@
+import { createNext } from 'e2e-utils'
+import { NextInstance } from 'test/lib/next-modes/base'
+import { fetchViaHTTP } from 'next-test-utils'
+
+describe('reading request body in middleware', () => {
+  let next: NextInstance
+
+  beforeAll(async () => {
+    next = await createNext({
+      files: {
+        'pages/_middleware.js': `
```

Comment:
Can we add a test case where the middleware reads the body but then continues to an API route that returns the body from [`req.body`](https://nextjs.org/docs/api-routes/api-middlewares)?

Reply:
good idea